python/docs/source/development/debugging.rst

```diff
@@ -341,7 +372,12 @@ Python/Pandas UDF
 ~~~~~~~~~~~~~~~~~
 
 To use this on Python/Pandas UDFs, PySpark provides remote `Python Profilers <https://docs.python.org/3/library/profile.html>`_ for
-Python/Pandas UDFs, which can be enabled by setting ``spark.python.profile`` configuration to ``true``.
+Python/Pandas UDFs.
+
+SparkContext-based
```

Comment:
I think you can just leave this as are, and just add one additional section called runtime profiler

Reply:
cc @ueshin do you have other thoughts?

Reply:
How about put the new doc to the first place?

- Identifying Hot Loops (Python Profilers)
    - Driver Side
    ...
    - Executor Side
        - Python/Pandas UDF
        Show the new profiler usage
        - Legacy (for RDD or non-Spark Connect)
        Put the current doc here

Reply:
We will remove "legacy" profilers for readability and clarity and start preparing migration guide.

Reply:
I have a workaround [76e7387](https://github.com/apache/spark/pull/45269/commits/76e738768d591ff59e9f14210bbccecc9458896a) by using autoclass, but it doesn't look consistent with the rest of the page, as shown below.

![image](https://github.com/apache/spark/assets/47337188/ab2e8f7b-2d65-4788-af47-354bd66a6fa2)

I'm wondering if we should have a follow-up designated for that part.